src/EPR.RegulatorService.Facade.Core/Services/Regulator/RegulatorOrganisationService.cs

```diff
@@ -64,62 +67,69 @@ public async Task<Result<CreateRegulatorOrganisationResponseModel>> CreateRegula
                     {
                         string content = await createdOrganisation.Content.ReadAsStringAsync();
 
-                        _logger.LogInformation($"Create regulator organisation name {request.Name} service content response is {content}");
```

Comment:
The standard way to use Microsoft logger in other project is as follows
 _logger.LogInformation($"Create regulator organisation name {Name} service content response is {Content}", request.Name, content);